concourse-monitoring/templates/prometheusrules.yaml

```diff
@@ -0,0 +1,53 @@
+apiVersion: monitoring.coreos.com/v1
+kind: PrometheusRule
+metadata:
+  name: {{ include "concourse-monitoring.fullname" . }}
+  labels:
+{{ include "concourse-monitoring.labels" . | indent 4 }}
+    {{- range $key, $val := .Values.prometheusSelectorLabels }}
+    {{ $key }}: {{ $val | quote }}
+    {{- end}}
+  name: "{{ .Release.Name }}"
+  namespace: "{{ .Release.Namespace }}"
+spec:
+  groups:
+    - name: concourse-monitoring.rules
+      rules:
+        - alert: ConcourseRunningWorkersMismatch
+          expr: concourse_workers_registered{state="running"} != {{ .Values.concourse_workers_desired }}
```

Comment:
Still not super-happy about hard-coding the number of workers like this, but I don't see any other way at the moment

Reply:
changed that to alert on stale workers as discssed